### PR TITLE
added more context to public team visibility

### PR DESCRIPTION
**Pull request by anwara-ms**
added more context to public team visibility, based on customer query.

### CODE REVIEW

**Comment by anwara-ms** [comment]
got more feedback on this (see below). I just want to highlight that when we say teams are not visible, we mean all teams. You can remove "public"

> "Let’s not use the term “Public” here (we will enable private team discovery as well outside of scoped/IB). I don’t think documentation text needs to be changed, you can highlight that by putting in a box perhaps?"

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-scoped-directory-search.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ appliesto:
...
 
-Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Information Barrier policies](https://docs.microsoft.com/microsoft-365/compliance/information-barriers) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover teams when scoped search is in effect. 
+Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Information Barrier policies](https://docs.microsoft.com/microsoft-365/compliance/information-barriers) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover any public teams when scoped search is in effect, but existing memebers in those public teams can add users, as allowed by active IB policies.
```
Comment (JohanFreelancer9) [inline Teams/teams-scoped-directory-search.md:29]:
```suggestion
Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Information Barrier policies](https://docs.microsoft.com/microsoft-365/compliance/information-barriers) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover any teams when scoped search is in effect, but existing members in those teams can add users, as allowed by active Information Barrier policies.
```

Comment (JohanFreelancer9) [reply Teams/teams-scoped-directory-search.md:29]: Two minor edits, and also removed the word "public" as per feedback.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by anwara-ms — COMMENTED** [review]
looks good. I dont have approve rights. Please proceed with publishing this change.

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
is this rolled out yet? please push it to prod

**Comment by anwara-ms** [comment]
Hi. I see that this is still not published. Can we get it out the door fast plz

Thank you!
 --
Anwar Ali | PM @ TEAMS | LinkedIn<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.linkedin.com%2Fin%2Fanwarali&data=04%7C01%7CAli.Anwar%40microsoft.com%7C61a24ee754db4b1493e408d6b4dc7b41%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636895256392090042%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C-1&sdata=KpQ4z1jxZOc8s8IWXAYS3oVAbqKpT94E%2BzEB%2FLypv2c%3D&reserved=0>
IP, Security and Compliance Team

**Comment by anwara-ms** [comment]
Hi, I see that the changes are not live yet.

image

**Pull request merged**